src/test/java/core/basesyntax/model/TransactionTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class TransactionTest {
```

Comment:
we can remove this test, because generally we are just testing business logic (services), and not models